lib/Doctrine/Query.php

```diff
@@ -449,7 +449,7 @@ public function processPendingFields($componentAlias)
                             . " must have at least one field selected.");
                 }
             }
-            return;
+            return null;
```

Comment:
Since this might return **something** (a string), changing this to return null instead of void.